.github/workflows/qa-checks.yml

```
@@ -1,6 +1,6 @@
 name: OSP Web QA Checks
 
-on: [push, pull_request]
+on: [push, pull_request_target]
```

```suggestion
on: [push, pull_request]
```
No need for pull_request_target

---

Instead of this please change the method to use wearerequired/lint-action@v1
because it fails on Prs from the fork.

---

@codesankalp I think wearerequired/lint-action@v1 is already using see line no. [24](https://github.com/anitab-org/anitab-forms-web/blob/577e671a97605de2ea35706e62859b3fcfbb6fca/.github/workflows/qa-checks.yml#L24).

---

> Instead of this please change the method to use wearerequired/lint-action@v1
> because it fails on Prs from the fork.

@codesankalp checks are still failing?